src/Codeception/Command/Run.php

```diff
@@ -404,29 +430,30 @@ public function execute(InputInterface $input, OutputInterface $output)
             $suites = $suite ? explode(',', $suite) : Configuration::suites();
             $this->executed = $this->runSuites($suites, $this->options['skip']);
 
-            if (!empty($config['include']) and !$suite) {
-                $current_dir = Configuration::projectDir();
+            if (!empty($config['include']) && !$suite) {
+                $currentDir = Configuration::projectDir();
                 $suites += $config['include'];
-                $this->runIncludedSuites($config['include'], $current_dir);
+                $this->runIncludedSuites($config['include'], $currentDir);
             }
 
             if ($this->executed === 0) {
-                throw new \RuntimeException(
+                throw new RuntimeException(
                     sprintf("Suite '%s' could not be found", implode(', ', $suites))
                 );
             }
         }
 
         $this->codecept->printResult();
 
-        if (!$input->getOption('no-exit')) {
-            if (!$this->codecept->getResult()->wasSuccessful()) {
-                exit(1);
-            }
+        if (!$input->getOption('no-exit') && !$this->codecept->getResult()->wasSuccessful()) {
+            exit(1);
         }
         return 0;
     }
 
+    /**
+     * @return mixed|void
```

Comment:
However, I had to add return null; explicitly, because otherwise, CI jobs would fail:
`PHP Fatal error:  Uncaught TypeError: Codeception\Command\Run::matchSingleTest(): Return value must be of type ?array, none returned in /home/runner/work/Codeception/Codeception/src/Codeception/Command/Run.php:487`

Reply:
However, I had to add `return null`; explicitly, because otherwise, CI jobs would fail:
`
PHP Fatal error:  Uncaught TypeError: Codeception\Command\Run::matchSingleTest(): Return value must be of type ?array, none returned in /home/runner/work/Codeception/Codeception/src/Codeception/Command/Run.php:487
`